base/src/main/java/io/spine/validate/EnumFieldValidator.java

```diff
@@ -34,7 +37,7 @@
      *         the value to validate
      */
     EnumFieldValidator(FieldValue fieldValue) {
-        super(fieldValue, false, true);
+        super(fieldValue, false, true, ImmutableSet.of());
```

Comment:
@serhii-lekariev, please add a simple `super` constructor, so you don't have to repeat `false, true, ImmutableSet.of()` every time. The existing one should then only be used for customization.